src/main/java/com/exadel/discount/service/DiscountServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.discount.service;
+
```

Comment:
I think we should keep single project structure. For example, all impl in impl package. But now we have some in impl, some out of it. The same is about interfaces package.

Reply:
Yes, I already changed structure in my project but don't push yet.